docs/authorization_request.rst

```diff
@@ -0,0 +1,92 @@
+.. _authorization_request:
+
+
+Authorization request
+---------------------
+
+Authorization request provides way how authorize insecure user operations.
+
+Create authorization request
+----------------------------
+
+To create authorization request you can use ``auth_token.utils.create_authorization_request`` function. Function create new authorization request which can be authorized via specified method. Right now there are two authorization methods OTP or mobile device. OTP can be authorized with knowledge of secret key (code). Mobile device authorization can be done via device ID and secret device token stored in ``MobileDevice`` model.
+
+
+.. function:: create_authorization_request(type, slug, title, description=None, authorization_token=None, related_objects=None, data=None, otp_key_generator=None, otp_sender=None, mobile_device=None, expiration=None)
```

Comment:
I would not hardcode this in the docs. Nobody's gonna update it in case of changes. Why don't you use autogenerated reference documentation from docstrings?

Reply:
I can try it. But the library is not set on it right now

Reply:
It should not be that hard, just add Napoleon which understands google style docstrings:
https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html

Then use it in Sphinx configuration:
https://github.com/druids/django-pynotify/blob/master/docs/conf.py#L42

And the reference docs page looks like this:
https://github.com/druids/django-pynotify/blob/master/docs/reference.rst

And rendered:
https://django-pynotify.readthedocs.io/en/latest/reference.html

Reply:
This is future request